lib/commands/menu/set_menu.rb

```diff
@@ -55,7 +54,7 @@ def extract_url(request)
     end
 
     def save_menu(url)
-      Menu.new(url: url, date: Time.now, office: @apprentice.office).save
+      Menu.new(url: url, date: Time.now, office: @user.office).save
```

Comment:
It would be good to move this to the Menu Modal as the responsibility of this class shouldn't be to save in the database. Something like Menu.picked_up(url, office)

Reply:
I agree